fe2-android/app/src/test/java/com/github/dedis/popstellar/model/network/method/message/data/election/ElectionQuestionTest.java

```diff
@@ -20,7 +20,7 @@
   private final List<String> allMethods = Arrays.asList("Plurality", "Plurality");
   private final List<String> allQuestions = Arrays.asList("Question", "Question2");
   private final List<Boolean> allWriteIns = Arrays.asList(false, false);
-  private final ElectionSetup electionSetup = new ElectionSetup(name, time, time, allMethods, allWriteIns,
+  private final ElectionSetup electionSetup = new ElectionSetup(name, time, time, time, allMethods, allWriteIns,
```

Comment:
`time, time, time` sounds bizarre. Naive question: could we find another, more elegant way to initialise this object ?